[14.0][ADD] delivery_mrw

mariadforgeflow opened the pull request:
New module `delivery_mrw`. 
Done following the SAGEC API documentation provided by MRW.

Commit:
[ADD] l10n_es_aeat_mod130: Módulo para la declaración del modelo 130 - IRPF pago fraccionado estimación directa

Commit:
Launchpad automatic translations update.

Commit:
[FIX] l10n_es_aeat_mod130: Corrección del archivo exportado para evitar código 0339 en la presentación.

acysos committed:
[FIX] l10n_es_aeat: compatibilidad de la función export_to_boe para el modelo 340

Commit:
[FIX] l10n_es_aeat_mod130: Tilde en archivo que hace fallar runbot.

Commit:
[FIX] l10n_es_aeat_mod130/mod303: import corregido para que valga también para la v8.

rlizana committed:
[MIG] l10n_es_aeat_mo130

* Clases a CapWords
* PEP8
* Icono del modulo

pedrobaeza committed:
[FIX] l10n_es_aeat_mod130: Problema en la búsqueda cuando los periodos no son trimestrales + PEP8 79 cols

Commit:
[IMP] unificar campos en el módulo base + migrar a la nueva api.

pedrobaeza committed:
[FIX+IMP] l10n_es_aeat_*: PEP8 + Refinamientos varios

oca-transbot committed:
OCA Transbot updated translations from Transifex

hugosantosred committed:
[IMP] Añadir reglas de registro multicompañía a informes financieros e informes AEAT

Commit:
[MIG] l10n_es_aeat_mod130: Migration to 11.0

pedrobaeza committed:
[MIG] l10n_es_aeat_mod130: Finish migration

- Remove redundant fields
- Expenses of difficult justification
- Monetary type in fields
- Adjust BOE export format
- Adjust views
- Test BOE export
- Other fine tuning

ernestotejeda committed:
[MIG] l10n_es_aeat_mod130: Migration to 12.0

CarlesAntoli committed:
Translated using Weblate (Spanish)

Currently translated at 51.4% (73 of 142 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod130/es/

rlizana committed:
[12.0] l10n_es_aeat_*: [FIX] Caracteres no válidos 'NIF Empresa Desarrollo'

rlizana committed:
[ADD] delivery_seur

[FIX] rename views to  && [FIX] small fixes

[UPD] remove depend to delivery_price_method

[UDP] change depend stock_picking_delivery_info_computation to delivery_package_number

[FIX] work with latin characters

[UPD] test replace to SavepointCase

[UPD] test replace to setUpClass

[UPD] force carrier margin to 0

[UPD] test create picelist

[REM] set a margin value

[UPD] .rst files

[UPD] README.rst

javierobcn committed:
[IMP] l10n_es_aeat_mod130: Add chatter

oca-travis committed:
[UPD] Update delivery_seur.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

rlizana committed:
[UPD] delivery_seur: replace field weight to shipping_weight for send expedition

[UDP] delivery_seur: information for create shipping in 'prepare' method for ease overwrite

[UPD] delivery_seur: add comment

[UPD] comments

oca-travis committed:
[UPD] Update delivery_seur.pot

delivery_seur 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

oca-travis committed:
[UPD] Update delivery_seur.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

chienandalu committed:
[FIX] delivery_seur: Cash on Delivery value

- Currently there's no support for Cash on Delivery expeditions. We
shouldn't give any value to the Cash on Delivery field as it's recorded
in the expedition and can lead to errors.

delivery_seur 12.0.1.0.2

chienandalu committed:
[FIX] delivery_seur: pesoBulto

According to SEUR, the value expected is the weight by package not the
total of the expedition.

chienandalu committed:
[IMP] delivery_seur: improve traceability and error catch

- Add easy traceability of communications to SEUR with last sent and
received data.
- Better exception handling as in some case we didn't have a different
error response.

chienandalu committed:
[FIX] delivery_seur: avoid dead locks

chienandalu committed:
[FIX] delivery_seur: full shipping addrees name

Follow the same format as in the delivery slip

[UPD] Update delivery_seur.pot

delivery_seur 12.0.1.1.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

chienandalu committed:
[IMP] delivery_gls and delivery_seur: add informative exceptions

Non implemented methods should raise informative exceptions if used

[UPD] Update delivery_seur.pot

delivery_seur 12.0.1.1.2

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

chienandalu committed:
[FIX] delivery_seur: zebra label

SEUR sends the Zebra label with an CI10 command, wich stands for spanish
character set encoding. We must encode the resultant txt file with
iso-8859-15 as well to support accents properly.

[UPD] Update delivery_seur.pot

delivery_seur 12.0.1.1.3

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-12.0/l10n-spain-12.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-delivery_seur/

javiermatos committed:
Setting field 06 for Model 130 to editable. Computing initial value for 06 field using withholdings.

brendapaniagua committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (142 of 142 strings)

Translation: l10n-spain-12.0/l10n-spain-12.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-12-0/l10n-spain-12-0-l10n_es_aeat_mod130/ca/

pedrobaeza committed:
[FIX] delivery_seur: Proper signature for seur_rate_shipment

Although it will raise another exception, at least it will show the
proper message and not a traceback.

Fixes #1605

victoralmau committed:
[IMP] delivery_seur: black, isort, prettier

victoralmau committed:
[MIG] delivery_seur: Migration to 13.0

oca-travis committed:
[UPD] Update delivery_seur.pot

OCA-git-bot committed:
[UPD] README.rst

brendapaniagua committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (71 of 71 strings)

Translation: l10n-spain-13.0/l10n-spain-13.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-delivery_seur/es/

brendapaniagua committed:
Added translation using Weblate (Catalan)

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (68 of 68 strings)

Translation: l10n-spain-13.0/l10n-spain-13.0-delivery_seur
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-13-0/l10n-spain-13-0-delivery_seur/ca/

cubells committed:
[Fix] delivery_seur: avoid error due to non existent field

OCA-git-bot committed:
delivery_seur 13.0.1.0.1

cubells committed:
[IMP] delivery_seur: missing product codes and fix id_mercancia bug

oca-travis committed:
[UPD] Update delivery_seur.pot

OCA-git-bot committed:
delivery_seur 13.0.1.0.2

victoralmau committed:
delivery_seur: Code refactor

oca-travis committed:
[UPD] Update delivery_seur.pot

OCA-git-bot committed:
delivery_seur 13.0.1.1.0

chienandalu committed:
[FIX] delivery_seur: partner name

If we provide the display name of the partner, there's a high chance
that it's trimmed in the label leading to confusion.

This way, we can keep the relevant data considering if the contact is a
delivery address, a child contact or a commercial partner.

TT33632

chienandalu committed:
[FIX] delivery_seur: label filename

In v12, with the parameter datas_fname we could force the file to be
downloaded as such filename. Now that parameter doesn't exist and the
filename format is lost from the previous version.

odoo/odoo@c212cfe8992292f57acacd809de26e1817533095

This is specilly problematic with txt format and some printer which
doesnt support joliet filenames or even files without extension.

The mimetype for text files was also wrong, as 'application/txt' doesn't
exist. See Mozilla Dev docs for more info on this:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

TT34599

OCA-git-bot committed:
delivery_seur 13.0.1.1.2

sergio-teruel committed:
[FIX] l10n_es_aeat: Don't error on manually created export lines

OCA-git-bot committed:
[UPD] README.rst

victoralmau committed:
[IMP] delivery_seur: TestDeliverySeurBase added to improve inherit.
TT37147

OCA-git-bot committed:
delivery_seur 13.0.1.1.3

ao-landoo committed:
[IMP] l10n_es_dua_ticketbai_batuz: black, isort, prettier

victoralmau committed:
[FIX] delivery_seur: Set the correct weight if we use packages.
TT37180

ao-landoo committed:
[MIG] l10n_es_dua_ticketbai_batuz: Migration to 13.0

Commit:
[UPD] Update l10n_es_dua_ticketbai_batuz.pot

OCA-git-bot committed:
[UPD] README.rst

ao-landoo committed:
[IMP] l10n_es_dua_ticketbai_batuz: black, isort, prettier

ao-landoo committed:
[MIG] l10n_es_dua_ticketbai_batuz: Migration to 14.0

mariadforgeflow committed:
[ADD] delivery_mrw

miquelalzanillas committed:
[MIG] l10n_es_ticketbai_pos: Migration to 14.0

victoralmau committed:
[IMP] l10n_es_aeat_sii_oca: Avoid modifying third-party number when an invoice has already been sent to the SII + Tests.
[FIX] l10n_es_aeat_sii_oca: Fix incorrect invoice type (in refund) to in_refund
TT38248

OCA-git-bot committed:
Merge PR #2445 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_sii_oca.pot

OCA-git-bot committed:
l10n_es_aeat_sii_oca 14.0.2.3.1

OCA-git-bot committed:
[UPD] addons table in README.md

miquelalzanillas committed:
[14.0][FIX] l10n_es_ticketbai: Remove unused library function calls

victoralmau committed:
[IMP] l10n_es_aeat_sii_oca: Avoid modifying customer invoice name when an invoice has already been sent to the SII + Tests + Refactor exceptions SII.
TT38286

ao-landoo committed:
[FIX] l10n_es_ticketbai: Get company IRPF taxes correctly

OCA-git-bot committed:
Merge PR #2447 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_sii_oca.pot

OCA-git-bot committed:
l10n_es_aeat_sii_oca 14.0.2.3.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2446 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 14.0.2.6.5

OCA-git-bot committed:
[UPD] addons table in README.md

Shide committed:
[FIX] l10n_es_aeat: _parse_aeat_vat_info - show correct aeat identification

- Tests for aeat_identification_type and aeat_identification
- Adds to ormcache those fields
- If user has aeat_identification_type, use aeat_identificationç

[FIX] l10n_es_aeat: Fix test_parse_vat_info_aeat_identification tests

OCA-git-bot committed:
Merge PR #2448 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 14.0.2.6.6

OCA-git-bot committed:
[UPD] addons table in README.md

xAdrianC-Kernet committed:
[FIX] l10n_es_ticketbai_pos: Fix tbai file generation with operation key different to 01 when this module is installed

OCA-git-bot committed:
Merge PR #2455 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat 14.0.2.2.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2417 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_dua_ticketbai_batuz.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #2457 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_ticketbai_pos.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #2119 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_payment_order_confirming_sabadell.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

victoralmau committed:
[IMP] delivery_seur: Add seur_use_packages_from_picking field in carrier to use (or not) package from pickings.
TT37180

Commit:
[UPD] Update delivery_seur.pot

OCA-git-bot committed:
delivery_seur 13.0.1.2.0

Vicenteagf committed:
[IMP] : black, isort, prettier

Vicenteagf committed:
[MIG] delivery_seur: Migration to 14.0

Update delivery_seur/models/seur_request.py

Co-authored-by: Víctor Martínez <victoralmau@users.noreply.github.com>

[FIX] delivery_seur: seur_use_packages_from_picking carrier field not needed anymore.

victoralmau committed:
[FIX] l10n_es_facturae_face: Change to the correct field name to fix tests.

OCA-git-bot committed:
Merge PR #2473 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_facturae_face 14.0.1.2.2

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2472 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update delivery_seur.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

olatzavanzosc committed:
Added translation using Weblate (Spanish)

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_dua_ticketbai_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_dua_ticketbai_batuz/es/

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (5 of 5 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_dua_ticketbai_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_dua_ticketbai_batuz/eu/

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (129 of 129 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai/es/

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (129 of 129 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai/eu/

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/es/

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (258 of 258 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/eu/

olatzavanzosc committed:
Added translation using Weblate (Spanish)

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (88 of 88 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api_batuz/es/

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (88 of 88 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api_batuz/eu/

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (75 of 75 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_batuz/es/

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (82 of 82 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_batuz
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_batuz/eu/

olatzavanzosc committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (37 of 37 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_pos
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_pos/es/

olatzavanzosc committed:
Added translation using Weblate (Basque)

olatzavanzosc committed:
Translated using Weblate (Basque)

Currently translated at 100.0% (41 of 41 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_pos
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_pos/eu/

DS-Landoo committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat/es/

xAdrianC-Kernet committed:
[FIX]vat regime key now takes code (str 2 digits) instead of id(int)

pedrobaeza committed:
[FIX] payment_redsys: Use updated library

pycrypto is unmaintained since several ago, and in fact pycryptodome was
the library being used in previous version. The API we are using is
compatible between both libraries though.

There's no problem for already deployed systems, as the check on the
external dependencies is only done at install.

We took the occasion to remove the try mechanism that is not needed now,
although the linter emits a warning.

etobella committed:
[FIX] l10n_es_facturae_face: Fix changes applied on EDI

OCA-git-bot committed:
Merge PR #2477 into 14.0

Signed-off-by etobella

OCA-git-bot committed:
l10n_es_facturae_face 14.0.1.2.3

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2476 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
payment_redsys 14.0.2.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

pedrobaeza committed:
[FIX] l10n_es_aeat: Consider correctly French overseas

For French overseas territories
(https://en.wikipedia.org/wiki/Overseas_France), there are no
intracommunity fiscal rules, but the country code that AEAT requires
should be FR, as it can be seen here:

https://sede.agenciatributaria.gob.es/Sede/todas-gestiones/impuestos-tasas/declaraciones-informativas/modelo-347-decla_____racion-anual-operaciones-personas_/instrucciones/codigos-paises-territorios.html

Thus, we need to call the mapping function differently if you want to
convert `EL` to `GR` for getting the European countries than to pass
the proper country code.

This is done adding an extra parameter for saying if we want to include
such replacements, and only being executed when the check about European
country is rejected, on contrary than with Greece (EL), that should be
included as intracommunity.

The country code is now also obtained from the vat itself if the first
2 letter belongs to any country code.

TT38835

etobella committed:
[FIX] l10n_es_Facturae_face: Force webservice backend to have an auth_type

OCA-git-bot committed:
Merge PR #2484 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_facturae_face 14.0.1.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

dalonsod committed:
[FIX] l10n_es_aeat_sii_oca: admit E3 export exempt cause

* Before this fix, when an invoice fiscal position is marked as Export, SII
  exempt cause is currently forced to E2.
* E3 exempt cause should also be considered in this situation. This commit fix
  it. E2 is still selected as default in other cases, anyway.

OCA-git-bot committed:
Merge PR #2474 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_ticketbai_pos 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2486 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat 14.0.2.2.2

OCA-git-bot committed:
[UPD] addons table in README.md

xAdrianC-Kernet committed:
[IMP] l10n_es_ticketbai: Include new tax 5%

xAdrianC-Kernet committed:
[IMP] l10n_es_ticketbai_batuz: Include new tax 5%

OCA-git-bot committed:
Merge PR #2496 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 14.0.2.7.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2497 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai_batuz 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

mariadforgeflow committed:
[IMP] delivery_mrw: remove department map model

LoisRForgeFlow reviewed (approved):
Fucntional review, all good! :+1:

ao-landoo committed:
[FIX] l10n_es_ticketbai: Make reversed_entry_id field not readonly on invoices

victoralmau commented:
Gracias por la aportación, ¿habéis visto https://github.com/OCA/l10n-spain/pull/2390?

Creo que lo mejor en este caso es fusionar el otro pr (después de las revisiones correspondientes), traerlo a 14.0 y si corresponde añadir lo que pudiera faltar. Tu que opinas @pedrobaeza ?

pedrobaeza commented:
Efectivamente, el otro PR debería tener prioridad si está correcto y es el mismo alcance.

OCA-git-bot committed:
Merge PR #2504 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai 14.0.2.7.1

OCA-git-bot committed:
[UPD] addons table in README.md

mariadforgeflow commented:
Hola @pedrobaeza @victoralmau! He revisado el otro módulo y puesto en un comentario todos los aspectos que faltan/fallan. He dejado a criterio del autor si prefiere añadirlo todo y migrar o quedarnos con este y hacer forward port.

OCA-git-bot committed:
Merge PR #2508 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat_sii_oca 14.0.2.3.3

OCA-git-bot committed:
[UPD] addons table in README.md

jabelchi committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/es/

jabelchi committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (37 of 37 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_pos
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_pos/es/

pedrobaeza committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/es/

RabbitJon-S73 committed:
[IMP] l10n_es_aeat_partner_check: add new selection

etobella committed:
[IMP] l10n_es_facturae: Allow to use invoice addresses

enriquemartin committed:
[14.0][FIX] l10n_es_dua_ticketbai_batuz - hacer compatible con módulo l10n_es_dua_sii

OCA-git-bot committed:
Merge PR #2512 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_dua_ticketbai_batuz 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

enriquemartin committed:
[14.0][FIX] l10n_es_ticketbai* - corregir problemas en entornos multicompany

LoisRForgeFlow committed:
[FIX] delivery_mrw: separador decimal en el campo peso.

victoralmau committed:
[FIX] l10n_es_account_statement_import_n43: Fix tests (required from https://github.com/OCA/bank-statement-import/pull/482/commits/8987b4c9937136f5db587d2bc442b92efdbddb29)

OCA-git-bot committed:
Merge PR #1937 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat 14.0.2.2.3

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2519 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_account_statement_import_n43 14.0.1.0.3

OCA-git-bot committed:
[UPD] addons table in README.md

mariadforgeflow committed:
[FIX] delivery_mrw: mrw_address computation

cubells committed:
[FIX][14.0] l10n_es_vat_book: missing 5% tax

OCA-git-bot committed:
Merge PR #2529 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_vat_book 14.0.2.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2514 into 14.0

Signed-off-by ao-landoo

OCA-git-bot committed:
l10n_es_ticketbai_batuz 14.0.1.1.1

OCA-git-bot committed:
l10n_es_ticketbai 14.0.2.7.2

OCA-git-bot committed:
l10n_es_ticketbai_api 14.0.2.3.4

OCA-git-bot committed:
[UPD] addons table in README.md

cubells committed:
[FIX][14.0] l10n_es_vat_book: add missing tax in vat book
- Added tax 'Iva soportado no sujeto (bienes)'

OCA-git-bot committed:
Merge PR #2532 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_vat_book 14.0.2.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

xAdrianC-Kernet committed:
[FIX] *: replace tabs for spaces

OCA-git-bot committed:
Merge PR #2533 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_facturae 14.0.2.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2538 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_aeat 14.0.2.3.0

OCA-git-bot committed:
[UPD] addons table in README.md

jabelchi committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/es/

mariadforgeflow committed:
[FIX] delivery_mrw: minor fixes

mariadforgeflow committed:
[IMP] delivery_gls_asm: hide cancel button for other carriers

OCA-git-bot committed:
Merge PR #2546 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
delivery_gls_asm 14.0.1.5.0

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #2510 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat_partner_check.pot

OCA-git-bot committed:
l10n_es_aeat_partner_check 14.0.1.2.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat_partner_check
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat_partner_check/

pedrobaeza committed:
[IMP] l10n_es_aeat_mod130: pre-commit stuff

pedrobaeza committed:
[MIG] l10n_es_aeat_mod130: Migration to 14.0

- Standard procedure
- Switch to computed fields
- AEAT base adaptation
- Tests adaptation

OCA-git-bot committed:
Merge PR #2552 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update l10n_es_aeat.pot

Commit:
[UPD] Update l10n_es_aeat_mod130.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat/

victoralmau committed:
[IMP] delivery_dhl_parcel: Update documentation

TT40002

victoralmau committed:
[FIX] delivery_dhl_parcel: Improve changes

Create _prepare_dhl_parcel_parcel_address_info() function to reduce code.

Improve receiver + sender information:
- Remove blank spaces on partner phone
- Set phone or mobile from partner
- Concatenate street and street2 field
- Limit address field to 40 characters

TT40002

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_aeat_mod130
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_aeat_mod130/

mariadforgeflow committed:
[FIX] delivery_mrw: separador decimal en el campo ImporteReembolso

OCA-git-bot committed:
Merge PR #2556 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
delivery_dhl_parcel 14.0.1.4.5

OCA-git-bot committed:
[UPD] addons table in README.md

carlosdauden committed:
[FIX] l10n_es_account_bank_statement_import_n43: Requerir 6 caracteres para buscar partners usando ilike

OCA-git-bot committed:
Merge PR #2560 into 14.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
l10n_es_account_statement_import_n43 14.0.1.0.4

OCA-git-bot committed:
[UPD] addons table in README.md

oihane committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (257 of 257 strings)

Translation: l10n-spain-14.0/l10n-spain-14.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-14-0/l10n-spain-14-0-l10n_es_ticketbai_api/es/

mariadforgeflow committed:
[IMP] delivery_mrw: add operation GetEnvios

mariadforgeflow committed:
[IMP] delivery_mrw: Update Readme

mariadforgeflow committed:
[IMP] delivery_mrw: update translations
